src/ochubryk/hometask3/count_consecutive_summers.py

```diff
@@ -0,0 +1,31 @@
+# Positive integers can be expressed as sums of consecutive
+# positive integers in various ways. For example, 42 can be
+# expressed as such a sum in four different ways:(a) 3+4+5+6+7+8+9,
+# (b) 9+10+11+12, (c) 13+14+15 and (d) 42. As the last solution (d)
+# shows, any positive integer can always be trivially expressed as
+# a singleton sum that consists of that integer alone.
+# Compute how many different ways it can be expressed
+# as a sum of consecutive positive integers.
+
+
+def count_consecutive_summers(num):
```

Comment:
one more interesting task is to find number of different sums of various (not consecutive) integers) and count it O-complexity

Reply:
It's too much for me according to my current skills so I'm going to try it later! :)